functions.php

```diff
@@ -39,18 +41,41 @@ function formatName(string $name): string
     return $name;
 }
 
-function validateInfo(string $mouseName,string $mouseBrand,string $mouseWeight): string
+function checkInputLength(string $mouseName, string $mouseBrand, int $mouseWeight): bool
+{
+    if ((strlen($mouseName) > 100) || (strlen($mouseBrand) > 50) || ($mouseWeight > 999) || ($mouseWeight < 1)) {
```

Comment:
because the if conditions return a boolean already, you can simplify this to just be:
```
return ((strlen($mouseName) > 100) || (strlen($mouseBrand) > 50) || ($mouseWeight > 999) || ($mouseWeight < 1));
```
that will return the true/false you need without the if